app/src/main/java/com/github/se/cyrcle/model/user/User.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.se.cyrcle.model.user
+
+data class User(
+    val userId: String = "",
```

Comment:
I do not see why we allow a default value for userId to be the empty string? shouldn't we forbid the creation of User Object if no userId is specified ?

Reply:
Agree with this, but I'm not sure it's too much of an issue at this point in time. As I said, we'll need to figure out what fields are mandatory or not but this is a design decision for a meeting.

Reply:
I kind of agree with Sean, this is kind of crucial for testing

Reply:
I strongly agree as well.